coders/composio_coders/cli.py

```diff
@@ -0,0 +1,147 @@
+import click
+import json
+import os
+import git
+from pathlib import Path
+
+from composio_coders.swe import CoderAgentArgs, CoderAgent
+
+MODEL_ENV_CONFIG_PATH = ".composio.coder.model_env"
+ISSUE_CONFIG_PATH = ".composio.coder.issue_config"
+KEY_GIT_ACCESS_TOKEN = "GITHUB_ACCESS_TOKEN"
+
+
+def get_git_root():
+    """Try and guess the git repo, since the conf.yml can be at the repo root"""
+    try:
+        repo = git.Repo(search_parent_directories=True)
+        path = Path(repo.git_dir)
+        repo_name = path.parent.name
+        return repo_name
+    except git.InvalidGitRepositoryError:
+        return None
+
+
+@click.command(name="setup", help="🔑 Setup model configuration in the current directory")
+def setup():
+    """Setup model configuration in the current directory."""
+    config = {}
+    model_env = click.prompt("🔑 Choose the model environment to be used for initiating agent", 
+                             type=click.Choice(['openai', 'azure'], case_sensitive=False))
+
+    if model_env == "openai":
+        config["model_env"] = "openai"
+        api_key = click.prompt("🔑 Please enter openai API key", type=str)
+        config["api_key"] = api_key
+    if model_env == "azure":
+        config["model_env"] = "azure"
+        api_key = click.prompt("🔑 Please enter azure key", type=str)
+        endpoint_url = click.prompt("🌐 Please enter Azure endpoint URL", type=str)
+        config["api_key"] = api_key
+        config["endpoint_url"] = endpoint_url
+
+    config_path = Path(MODEL_ENV_CONFIG_PATH)
+    with config_path.open('w') as f:
+        json.dump(config, f)
+    click.echo(f'🍀 Model configuration saved to {MODEL_ENV_CONFIG_PATH}')
```

Comment:
You can use `composio.storage.LocalStorage` class for storing/caching information locally

Reply:
added